android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt

```diff
@@ -80,15 +80,11 @@ class DDMFormView @JvmOverloads constructor(
 
 	override lateinit var formInstance: FormInstance
 
-	override val config = DDMFormViewConfig()
+    override lateinit var ddmFormListener: DDMFormListener
```

Comment:
I believe this should be optional, it's not mandatory for the user to listen to the events.